src/main/java/com/github/manolo8/darkbot/extensions/features/decorators/InstructionProviderDecorator.java

```diff
@@ -13,7 +14,9 @@ public class InstructionProviderDecorator extends FeatureDecorator<InstructionPr
 
     @Override
     protected void load(FeatureDefinition<InstructionProvider> fd, InstructionProvider obj) {
-        if (obj instanceof Module && !(obj instanceof TemporalModule)) showInstructions(obj, fd.getName());
+        if (obj instanceof Module && !(obj instanceof TemporalModule)
+                && !HeroManager.instance.main.isRunning()
+        ) showInstructions(obj, fd.getName());
```

Comment:
I think the pop-up should still appear if the bot is running, but what should change the behavior is as a result of what the change is. If the user changed module i think it's good that they get the pop-up, but i agree if it's an automatic change it shouldn't be shown.
I'm not exactly sure how to properly detect if it was automatic or not, since plugins can be the ones changing the config themselves.

Reply:
Was added API method for changing config and this call suppressing showing instructions. Also was added default true setting property for suppress showing instructions in all cases.

Reply:
implement via checking config value